test/unit/search/search_configuration_test.cpp

```diff
@@ -17,7 +17,7 @@ class search_configuration_test : public ::testing::Test
 
 using test_types = ::testing::Types<seqan3::search_cfg::max_error_rate<>,
                                     seqan3::search_cfg::max_error<>,
-                                    seqan3::search_cfg::mode<seqan3::detail::search_mode_best>,
+                                    decltype(seqan3::search_cfg::reporting_best_hit),
```

Comment:
```suggestion
                                    decltype(seqan3::search_cfg::reporting_best_hit),
```

Some tests are failing here